freqtrade/optimize/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 import gzip
 import json
+import ujson
```

Comment:
if you use `import ujson as json` it's a dropin-replacement (and should not break tests).